CRM/Export/Form/Map.php

```diff
@@ -183,14 +183,14 @@ public function postProcess() {
 
     $mapperKeys = $params['mapper'][1];
 
-    $checkEmpty = 0;
-    foreach ($mapperKeys as $value) {
-      if ($value[0]) {
-        $checkEmpty++;
+    $mappedFields = [];
```

Comment:
Doesn't this param need to . be . called mappedKeys - or the . call to exportComponents lower down needs changing?

Reply:
Whoops. Fixed now.